src/sync/installation.ts

```diff
@@ -236,7 +236,7 @@ const sendPayloadToJira = async (task: TaskType, jiraClient, subscription: Subsc
 				});
 				break;
 			case "deployment":
-				await jiraClient.deployment.submit(jiraPayload, repository.id, {
+				await jiraClient.deployment.submit(jiraPayload, repository.id, repository.full_name, {
```

Comment:
I don't this this work. Can you please dig into the api the call the graphql?
I don't think `full_name` appaer in the graphql. Could be nameWithOwner
Try refer to https://docs.github.com/en/graphql/reference/objects#deploymentconnection and check

Reply:
Something like this
<img width="1267" alt="image" src="https://github.com/atlassian/github-for-jira/assets/105693507/ebd13212-fd11-4621-b3ef-5542deb50762">